src/parameterizations/vertical/MOM_diabatic_driver.F90

```diff
@@ -1201,14 +1234,20 @@ subroutine diabatic_ALE(u, v, h, tv, BLD, fluxes, visc, ADp, CDp, dt, Time_end,
   real :: htot(SZIB_(G)) ! The summed thickness from the bottom [H ~> m or kg m-2].
   real :: Kd_add_here    ! An added diffusivity [H Z T-1 ~> m2 s-1 or kg m-1 s-1].
   real :: Idt     ! The inverse time step [T-1 ~> s-1]
+  real :: g_Rho0       ! Gravity divided by the reference density for computing N2 [H S2 R-1 ~> m4 s-2 kg-1]
```

Comment:
As written below, `g_Rho0` has units of [L2 H-1 T-2 R-1 ~> m4 kg-1 s-2 or m7 kg-2 s-2].  This factor does not seem to me to be very reasonable for use in non-Boussinesq mode, for which it is much more natural to work with derivatives of specific volume rather than density derivatives.